azure-pipelines.yml

```diff
@@ -201,7 +201,7 @@ jobs:
   # only required when updating dependents
   - script: nbgv cloud -a -c
     condition: eq(variables['UPDATE_DEPENDENTS'], 'true')
-    displayName: Set Could Version
+    displayName: Set nbgv Cloud Version
```

Comment:
This step is actually setting the version that the pipeline will report. It uses nbgv to grab that version, is not setting anything on nbgv and there's no nbgv cloud...

Reply:
Done. (Changed to Set Pipeline Version)